lib/client.js

```diff
@@ -99,23 +100,34 @@ extend(Raven.prototype, {
       globalContext.extra = options.extra;
     }
 
+    this.onFatalError = this.defaultOnFatalError = function (err, sendErr, eventId) {
+      console.error(err.stack);
```

Comment:
`cannot read property 'stack' of undefined` <- this could stop us from exiting, right? whats the guarantees here around err?